web/docs/tql2/operators/from.md

```diff
@@ -3,3 +3,241 @@ sidebar_class_name: hidden
 ---
 
 # from
+
+Obtains events from an URI, inferring the source, compression and format.
+
+```tql
+from uri:str, [loader_args… { … }]
+from events…
+```
+
+:::tip Use `from` if you can
+The `from` operator is designed as an easy way to get data into Tenzir,
+without having to manually write the separate steps of data ingestion manually.
+:::
+
+## Description
+
+The `from` operator is an easy way to get data into Tenzir.
+It will try to infer the connector, compression and format based on the given URI.
+
+Alternatively, it can be used to create events from records.
+
+### `uri: str`
+
+The URI to load from.
+
+### `loader_args… (optional)`
+
+An optional set of arguments passed to the loader.
+This can be used to e.g. pass credentials to a connector:
+
+```tql
+from "https://example.org/file.json", headers={Token: "XYZ"}
+```
+
+### `{ … } (optional)`
+
+A pipeline that can be used if inference for the compression or format does not work
+or is not sufficient.
+
+Providing this pipeline
+disables the inference for the decompression and parsing format in order to avoid
+confusion.
+
+:::tip Only specify `{ … }` if you need it for data ingestion
+The `{ … }` argument exists for data ingestion purposes. In most situations,
+inference should be sufficient and the pipeline should not be required.
```

Comment:
You are using the term "data ingestion" and "data ingestion pipeline" for the pipeline argument. Maybe that's a bit confusing?